optuna_dashboard/ts/action.ts

```diff
@@ -29,20 +29,16 @@ export const actionCreator = () => {
   const { apiClient } = useAPIClient()
   const { enqueueSnackbar } = useSnackbar()
   const [studySummaries, setStudySummaries] =
-    useRecoilState<StudySummary[]>(studySummariesState)
+    useAtom<StudySummary[]>(studySummariesState)
```

Comment:
The type arguments are explicitly specified in the original code, but they are redundant. I think it's a good opportunity to remove them:
```suggestion
    useAtom(studySummariesState)
```